### PR TITLE
[Homework #6] - JS basic 1 by Santiago Pereira

### PR DESCRIPTION
**Pull request by Santiago-Pereira**
Hi. 
I wasn't able to complete tasks #16 and #14 cause although I tried several times I couldn't accomplish the result...

### CODE REVIEW

**Review by ilya-ischenko — COMMENTED** [review]
You need to format the code. Install modules with help of yarn command, turn on "format on save" in you code editor and hit save button in this file. Do this for all your homeworks, please.

File: homeworks/santiago-pereira/js-06/main.js [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+                operator === "+" ? (value1 + value2) :
+                operator === "/" ? value1 / value2 :
+                operator === "*" ? value1 * value2 : operator;        
+}
+
+// Task 3: http://www.codewars.com/kata/printing-array-elements-with-comma-delimiters
+ const characters = ["h","o","l","a"];
+function solution3(ele){
+   return ele.join(","); 
+}
```
[the remark this answers is not in the document]
Comment (Santiago-Pereira) [reply homeworks/santiago-pereira/js-06/main.js:26]: it means element, i should have used str i believe.

File: homeworks/santiago-pereira/js-06/main.js [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+  for (let i = 0; i < word.length; i++) {
+    if (word.split(word[i]).length - 1 > 1) {
+      result += ")";
+    } else {
+      result += "(";
+    }
+  }
+
+  return result;
+}
```
[the remark this answers is not in the document]
Comment (Santiago-Pereira) [reply homeworks/santiago-pereira/js-06/main.js]: sorry i forgot to save it into a new variable.

**Review by ilya-ischenko — COMMENTED** [review]
I can not approve it because you have changed **package-lock.json**, **package.json**, **yarn.lock** files inside root folder.

**Pull request closed without merging**